reanalysis/reanalysis_global.toml

```diff
@@ -41,23 +40,19 @@ default_timeout = 6000
 default_memory = 'highmem'
 
 [images]
-vep = 'vep:105.0'
-gatk = 'gatk:4.2.6.1'
-cpg_workflows = 'cpg_workflows:latest'
-hail = 'cpg_workflows:latest'
-picard = 'picard_samtools:v0'
-picard_samtools = 'picard_samtools:v0'
+vep = 'australia-southeast1-docker.pkg.dev/cpg-common/images/vep:105.0'
+gatk = 'australia-southeast1-docker.pkg.dev/cpg-common/images/gatk:4.2.6.1'
+cpg_workflows = 'australia-southeast1-docker.pkg.dev/cpg-common/images/cpg_workflows:latest'
+picard = 'australia-southeast1-docker.pkg.dev/cpg-common/images/picard:2.27.4'
+samtools = 'australia-southeast1-docker.pkg.dev/cpg-common/images/samtools:1.16.1'
+hail = 'australia-southeast1-docker.pkg.dev/cpg-common/images/cpg_workflows:latest'
```

Comment:
```suggestion
vep = 'australia-southeast1-docker.pkg.dev/cpg-common/images/vep:105.0'
cpg_workflows = 'australia-southeast1-docker.pkg.dev/cpg-common/images/cpg_workflows:latest'
hail = 'australia-southeast1-docker.pkg.dev/cpg-common/images/cpg_workflows:latest'
```

Reply:
Trimming this as AIP doesn't use many images - they were just copied in bulk from the template